modules/seqexec/web/server/src/main/scala/seqexec/web/server/http4s/WebServerLauncher.scala

```diff
@@ -232,7 +232,17 @@ object WebServerLauncher extends IOApp with LogInitialization with SeqexecConfig
       Resource.make(alloc)(free).map(ExecutionContext.fromExecutor)
     }
 
-    def engineIO(httpClient: Client[IO], guideConfigDb: GuideConfigDb[IO], collector: CollectorRegistry): Resource[IO, SeqexecEngine] =
+  // Override the default client config
+  val clientConfig = new DefaultAsyncHttpClientConfig.Builder()
+    .setMaxConnectionsPerHost(200)
+    .setMaxConnections(400)
+    .setRequestTimeout(5000) // Change the timeout to 5 seconds
+    .setThreadFactory(threadFactory(name = { i =>
+      s"http4s-async-http-client-worker-${i}"
+    }))
+    .build()
```

Comment:
Maybe could also be

```scala
val clientConfig = new DefaultAsyncHttpClientConfig.Builder(AsyncHttpClient.defaultConfig)
  .setRequestTimeout(5000) // Change the timeout to 5 seconds
  .build()
```

to avoid resetting the other values?

Reply:
Oh good idea